data/core/encyclopedia/geography.cfg

```diff
@@ -244,18 +244,12 @@ Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish
 [topic]
     id=far_south
     title= _ "Far South"
-    text= _ "A vast region south of Kerlath home to the Dunefolk Civilization. It is also inhabited by many local creatures, including the Jinn and Wyverns. Very little is known for certain about what lies here." + "
-
-" +
-        #po: Geography of the Far South
-        _ "<header>text='Notable cities:'</header>
-    • Serrul: A rich maritime trade city of the Dunefolk along the northern coast of the Bay of Clouds, and the largest city in the Sandy Wastes.
-    • Th’arwya: A Dunefolk city-state along the Bay of Clouds in the Ashlan. Some say it is among the largest cities of Irdya, and perhaps among the richest." + "
+    text= _ "A vast region south of Kerlath home to many myserious races, including the Jinn and Wyverns. Very little is known for certain about what lies here." + "
 
 " +
         #po: Geography of the Far South
         _ "<header>text='Notable land features:'</header>
     • Mountains of Peril: A tall and deadly mountain range in the Far South inhabited by bands of vicious barbarians and monsters.
-    • Sandy Wastes: A hot desert located southeast of Kerlath home to various Dunefolk cities and trading outposts, as well as nomadic tribes in between. The northern parts of this desert are home to many treacherous monsters.
-    • Ashland Desert: A hot, dry expanse of sandy desert south of the Sandy Wastes, spearated from it by a shallow stretch of sea called the Bay of Clouds. It is often simply referred to as the Ashland."
+    • Sandy Wastes: A vast hot and dry desert located southeast of Kerlath. It is considered an impassable barrier by Wesnothians. It is filled with very sparse vegetation conssiting of no kore than a few shrubs and cacti.
+    • Isanbar: A possibly legendary mountain in the Sandy Wastes, surrounded entirely by flatter desert. It is not part of any mountain range and its existence is doubted by many."
```

Comment:
As a native Anglophone, I think the 'possibly' sounds weird here. Better to drop it.

Reply:
Replaced with tall mountain. It already says its existence is doubted later on.